### PR TITLE
[Mission-Control] Added upgrade notes for upgrading to 3.x/4.x chart versions

### PR DESCRIPTION
**Pull request by rahulsadanandan**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/mission-control/UPGRADE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Upgrade from 1.x,2x to 3.X (Chart Versions)
+
+* To upgrade chart version to 3x, you must be on chart version 1.0.5 or above as described in https://github.com/jfrog/charts/blob/master/stable/mission-control/CHANGELOG.md.
+* Data other than your licenses, such as your service information and insight, will not be available after the upgrade.
+* JFrog Mission Control v4.x is only compatible with JFrog Artifactory v7.x. To upgrade, you must first install JFrog Artifactory 7.x.
+* To know more about upgrading mission control, please refer -> https://www.jfrog.com/confluence/display/JFROG/Upgrading+Mission+Control#UpgradingMissionControl-UpgradingfromVersion3.5.1to4.x
+
+**DOWNTIME IS REQUIRED FOR AN UPGRADE!**
+
+* Data export is done with a migration script jfmcDataExport.sh (available in the other installers. Please get the file from https://jfrog.com/download-jfrog-platform). You can obtain the file from the docker compose installer.
```
Comment (chukka) [inline stable/mission-control/UPGRADE_NOTES.md:13]: @rahulsadanandan Can we add this `jfmcDataExport.sh`  file  under chart ?

File: stable/mission-control/UPGRADE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# JFrog Mission-Control Chart Upgrade Notes
+This file describes special upgrade notes needed at specific versions
+
+## Upgrade from 1.x,2x to 3.X (Chart Versions)
```
Comment (chukka) [inline stable/mission-control/UPGRADE_NOTES.md:4]: @rahulsadanandan we may need to add `4.x charts` after https://github.com/jfrog/charts/pull/976 merge

File: stable/mission-control/UPGRADE_NOTES.md (hunk 1 above)
Comment (rahulsadanandan) [reply stable/mission-control/UPGRADE_NOTES.md]: Its now added under files directory in mission-control chart

File: stable/mission-control/UPGRADE_NOTES.md (hunk 2 above)
Comment (rahulsadanandan) [reply stable/mission-control/UPGRADE_NOTES.md]: Its done.

**Comment by chukka** [comment]
@danielezer  can you please review this

**Pull request merged**